tsconfig.json

```diff
@@ -55,5 +61,8 @@
     /* Experimental Options */
     // "experimentalDecorators": true,        /* Enables experimental support for ES7 decorators. */
     // "emitDecoratorMetadata": true,         /* Enables experimental support for emitting type metadata for decorators. */
-  }
+  },
+  "include": [
```

Comment:
Ideally this would be temporary, possibly convert to an exclude to rule out the things we don't want to type test rather than the current whitelist approach.